### DISCUSSION TITLE
deep link url parameters

### DISCUSSION THREAD

**Discussion by mohamed000123** [invertase/react-native-firebase#6612] [Q&A]
i am using firebase dynamic links
and i want to make a dynamic link via the Firebase console
i want to add a query parameter (user id) to my Deep link URL 
to be some thing like this >> https://myApp.page.link/Profile/?id
how can i do this please ?

**Comment by mikehardy** [comment]
I believe this is not react-native-firebase specific, and you will be best served by inquiring on stackoverflow - our API mirrors the firebase-js-sdk API here, and the firebase team offers support on that platform for generic firebase questions
